### DISCUSSION TITLE
Does selenide support automate desktop app ?

### DISCUSSION THREAD

**Discussion by teoke1234** [selenide/selenide#2759] [Q&A]
Hi team,

I have a requirement for automate web browser and verify results in desktop app. 
Does selenide can handle to interact with desktop app ?

Thanks

**Comment by asolntsev** [comment]
@teoke1234 Short answer: yes. But I personally haven't tried it. :) 

Long answer: it depends how exactly your desktop app is built (with which technology). 
* If it's built with Electron, then definitely it's possible. I know people do it (using Chromedriver which can drive Electron apps).
* If it's built with Swing, then definitely it's possible. See an example: https://github.com/framebassman/fest-selenide
* in case of other technologies, you would need to find a corresponding WebDriver implementation (example for Swing: https://github.com/jalian-systems/marathonv5)
* Maybe this library helps: https://www.testdevlab.com/blog/testui-ui-test-automation-for-all-platforms-with-appium-and-selenide
